scripts/packages.mjs

```diff
@@ -0,0 +1,82 @@
+import {existsSync, readFileSync, writeFileSync} from 'fs';
+import {resolve} from 'path';
+
+export const packageDirsNpmTag = [
+  'atomic',
+  'auth',
+  'bueno',
+  'headless',
+  'atomic-react',
+  'atomic-angular/projects/atomic-angular',
+  'quantic',
+];
+
+export const packageDirsSnyk = ['headless', 'atomic'];
+
+/**
+ * @typedef PackageDefinition
+ * @property {string} name
+ * @property {string} version
+ * @property {string} packageDir Relative to `/packages` (e.g., `atomic-angular/projects/atomic-angular`).
+ */
+
+/**
+ * @param {string} fullPath
+ * @returns {PackageDefinition}
+ */
+export function getPackageDefinitionFromPath(fullPath) {
+  const {name, version} = JSON.parse(readFileSync(fullPath).toString());
+  return {name, version, packageDir};
+}
+
+/**
+ * @param {string} packageDir E.g.: `samples/some-package`
+ * @returns {PackageDefinition}
+ */
+export function getPackageDefinitionFromPackageDir(packageDir) {
+  const fullPath = resolve('.', 'packages', packageDir, 'package.json');
+  if (!existsSync(fullPath)) {
+    throw `Could not find package at ${fullPath}.`;
+  }
+  return getPackageDefinitionFromPath(fullPath);
+}
+
+/**
+ * @param {string} packageName
+ */
+export function getPackageDefinitionFromPackageName(packageName) {
+  const packageDef = packageDirsNpmTag
+    .map(getPackageDefinitionFromPackageDir)
+    .find(({name}) => name === packageName);
+  if (!packageDef) {
+    throw `Could not find package "${packageName}".`;
+  }
+  return packageDef;
+}
+
+/**
+ * @param {string} packageName
+ * @param {string} newVersion
+ * @param {string[]} depdendenciesPackageDirs E.g.: [`samples/some-package`]
+ */
+export function updatePackageVersion(
+  packageName,
+  newVersion,
+  depdendenciesPackageDirs
+) {
+  depdendenciesPackageDirs.forEach((packageDir) => {
+    const fullPath = resolve('.', 'packages', packageDir, 'package.json');
+    const originalContentAsText = readFileSync(fullPath).toString();
+    let newContent = originalContentAsText.replace(
+      new RegExp(`("${packageName.replace('/', '\\/')}"\\s*:\\s*")([^"]*)`),
+      '$1' + newVersion
+    );
+    if (packageName === JSON.parse(originalContentAsText).name) {
+      newContent = newContent.replace(
+        /("version"\s*:\s*\")([^"]*)/,
+        '$1' + newVersion
+      );
+    }
+    writeFileSync(fullPath, newContent);
+  });
+}
```

Comment:
Why not parse, edit then, write? It'll be more maintainable than a RegExp nah?

Reply:
![yakuza-majima.gif](https://user-images.githubusercontent.com/12366410/202486219-bf94cc58-5974-408d-b426-af1f3c2e2ac1.gif)
https://github.com/sindresorhus/detect-indent